css/style.css

```diff
@@ -291,4 +291,63 @@ html {
   height: 55rem;
   border-radius: 100%;
   box-shadow: 0 3rem 5rem rgb(0, 0, 0, 0.3);
+}
+
+/******************************/
+/* INGREDIENTS SECTION */
+/******************************/
+
+.ingredients-section {
+  background: url('../gallery/banners/ingredients-banner.webp');
+  background: url('../gallery/banners/ingredients-banner.jpg');
+  background-size: cover;
+  width: 100%;
+  padding: 10rem 0;
+  display: flex;
+  justify-content: flex-end;
+}
+
+.dark-square {
+  width: 57rem;
+  background-color: rgba(0, 0, 0, 0.6);
+  padding: 7.6rem;
+  text-align: center;
+  margin-right: 21.5rem;
+}
+
+.slogan-ingredients-section {
+  font-family: "Yeseva One", sans-serif;
+  font-size: 3.6rem;
+  font-weight: 400;
+  color: #FFFFFF;
+  margin-bottom: 1.6rem;
+}
+
+.decoration-ingredients {
+  width: 25rem;
+  height: 2.4rem;
+  margin-bottom: 2.4rem;
+  opacity: 70%;
+}
+
+.description-ingredients-section {
+  font-family: 'Lato', sans-serif;
+  font-size: 1.6rem;
+  font-weight: 400;
+  color: #FFFFFF;
+  line-height: 2.4rem;
+  margin-bottom: 7rem;
+}
```

Comment:
Same here, looks like we already have such parameters (reusable styles)

Reply:
And in this case, too, there are slight differences in properties. Such as font color and padding.